### PR TITLE
Add in release-2.0 changes to master

### PR DESCRIPTION
**Pull request by jquick**
This change merges in all 2.0 changes into master.

Signed-off-by: Jared Quick <jquick@chef.io>

### CODE REVIEW

**Review by chris-rock — APPROVED** [review]
Awesome work!

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ sudo: required
 branches:
   only:
     - master
+    - release-2.0
```
Comment (jquick) [inline .travis.yml:5]: We should remove this from travis and appveyor

**Review by miah — APPROVED** [review]
Very exciting! Great work everybody <3

File: lib/inspec/schema.rb [hunk 2]
```diff
@@ -8,6 +8,31 @@ class Schema
       'additionalProperties' => false,
       'properties' => {
         'duration' => { 'type' => 'number' },
+        'controls' => {
```
Comment (jquick) [inline lib/inspec/schema.rb:11]: This can be removed.

**Review by jquick — COMMENTED** [review]
Found some things to remove but will likely do after as we don't want to restart travis during its issues. Lookin good!!

**Comment by jquick** [comment]
Overriding the DCO bot as it breaks at >~30 commits. All other tests have passed.

**Comment by dialex** [comment]
Where can I check the "release notes" of this major version 2.0?
I want to have an overview of what changed so that my team can decide if we should upgrade.

**Comment by jquick** [comment]
Hey @dialex  - This post has the changes listed https://discourse.chef.io/t/inspec-2-0-release/12612

**Comment by dialex** [comment]
YES! That's exactly what I was looking for. Thanks.
Tip: I think you should make it more explicit, I could not find it anywhere in InSpec's repo.

**Pull request merged**